src/SelectSingle.vue

```diff
@@ -131,8 +131,6 @@
 			},
 			updateMenuState(open: boolean, callFocus = true) {
 				this.open = open
-
-				if (!open) this.$emit('select', this.options[this.activeIndex])
```

Comment:
it used to cause doubled `select` event
![Uploading Screen Recording 2020-11-20 at 03.34.33 PM.gif…]()